src/components/AppTopBar.jsx

```diff
@@ -0,0 +1,49 @@
+import * as React from "react";
```

Comment:
Możecie w `tsconfig.json` ustawić `allowSyntheticDefaultImports` to nie będzie się ESlint pultał